server/container/guice/cassandra-guice/pom.xml

```diff
@@ -395,6 +395,26 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>no-docker-test</id>
+            <activation>
+                <property>
+                    <name>!env.DOCKER_HOST</name>
```

Comment:
No.

If I do not have a docker host environment variable but a valid docker installation, java code is smart enough to find it.

(You are breaking my dev environment)

I don't like also the idea that we should add a maven profile every time we will use docker in tests: we will forget and that gonna be a mess.

I would prefer a more dev friendly approach: Catching the illegal exception and maybe even checking the message or the cause, to ensure that we can not find the docker installation. Then simply skip the test.

Proposed hack (in SwarmContainer to be not invasive in caller code):

```
    @Override
    public void start() {
        try {
            super.start();
        } catch (IllegalStateExeption e) {
            if (/* e correspond to not found docker install. Check message or cause to check this */) {
                Assume.assumeTrue(false, "Skipping test as no Docker environment was found"); // only approache that allow to programmatically skip a test ?
            }
            throw e;
        }
    }
```